contest/templates/contest/contest.html

```diff
@@ -52,19 +52,20 @@ <h1 class="panel-title" style="text-align:center;">Time</h1>
         <h1 class="panel-title" style="text-align:center;">Clarification</h1>
       </div>
       <div id="contest_clarification">
-        <table class="table">
+        <table class="table" style="table-layout: fixed; width: 100%">
           <thead>
             <tr>
               {% if user|can_reply:contest %}
-                <th></th>
+                <th width="5%"></th>
               {% endif %}
-              <th>#</th>
-              <th>Problem</th>
-              <th>Asker</th>
-              <th>Description</th>
-              <th>Reply</th>
-              <th>Replier</th>
-              <th>Reply Time</th>
+              <th width="2%">#</th>
+              <th width="15%">Problem</th>
+              <th width="10%">Asker</th>
+              <th width="20%">Description</th>
+              <th width="20%">Reply</th>
+              <th width="10%">Replier</th>
+              <th width="10%">Reply Time</th>
+              <th width="5%">For all team</th>
```

Comment:
If it is set in setting, someone can set the layout like this.
In my opinion, this attribute is not something about personal preference,
such as theme, which is perfect to put in setting.